airflow/providers/amazon/aws/transfers/mongo_to_s3.py

```diff
@@ -49,8 +49,8 @@ class MongoToS3Operator(BaseOperator):
     :type mongo_db: str
     :param replace: whether or not to replace the file in S3 if it previously existed
     :type replace: bool
-    :param allow_disk_use: in the case you are retrieving a lot of data, you may have
-        to use the disk to save it instead of saving all in the RAM
+    :param allow_disk_use: enables writing to temporary files in the case you are handling large dataset
+        when running an aggregate pipeline.
```

Comment:
```suggestion
        This only takes effect when `mongo_query` is a list - running an aggregate pipeline
```

Reply:
@amatellanes  Can you also update line:43 to `:type mongo_query: Union[list, dict]`? Much appreciated!

Reply:
@xinbinhuang Both changes done :heavy_check_mark: